.github/workflows/k6-ci-loadtest-prod.yml

```diff
@@ -0,0 +1,47 @@
+name: k6 Load Test
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Is it the same as dev? It should be possible to specify two branches and avoid file duplication